track_info.c

```diff
@@ -52,16 +48,18 @@ struct track_info *track_info_new(const char *filename)
 	uint64_t uid = atomic_fetch_add_explicit(&cur_uid, 1, memory_order_relaxed);
 	BUG_ON(uid == 0);
 
-	struct track_info *ti;
-	ti = xnew(struct track_info, 1);
-	ti->uid = uid;
+	struct track_info_priv *priv = xnew(struct track_info_priv, 1);
+	atomic_store_explicit(&priv->ref_count, 1, memory_order_relaxed);
```

Comment:
It will do the job, right? As I understand it, we can't have any races here, so a simple assignment is good enough. Is there any reason to use explicit atomic instructions?